hashistack/part_2_hashistack/playbooks/inventory/hosts

```diff
@@ -17,3 +17,6 @@ server01.consul
 
 [nomad_deployer]
 server02.consul
+
+[nomad_instances]
```

Comment:
This is actually not needed, as the Nomad servers are listed under `docker_instances` (not very intuitive I know, I'm planning to change it in the hardened version)

Reply:
I wasn't able to run the Ansible Script without that line in the inventory because the Nomad role failed at the assert.yml where it checks if "nomad_instances" is in the groups.

Reply:
I have the same problem and have resolved it with adding `nomad_instances`